specs/HiddenPdfToolbarItems.md

```diff
@@ -0,0 +1,145 @@
+# Background
+When open a PDF file in Edge, there will be a toolbar at the top. The toolbar provide the functionality like print, save and annotation.
```

Comment:
Just a few minor text fixes
```suggestion
When opening a PDF file in Edge, there will be a toolbar at the top. The toolbar provides functionality like print, save and adding annotations.
```

Reply:
Some stupid faults. I will be more careful with syntax.